### ISSUE TITLE
Trying to setup using WordPress 4.1

### ISSUE THREAD

**Issue by cfletcher** [fukata/wp-flickr-press#33]
I'm trying to setup access to Flickr using version 2 of wp-flickr-press on Wordpress 4.1 but after I put my key and secret in and press update the popup window loads for a second or two then stays blank.
Chrome dev tools shows an error of:
Uncaught TypeError: Cannot read property 'nsid' of undefined                wp-admin/options-general.php?page=wp-flickr-press/FpAdminSettingEvent.php:328

**Comment by fukata** [comment]
Can you debug?

Please modify flickr_oauth_callback.php below.

```diff
diff --git a/flickr_oauth_callback.php b/flickr_oauth_callback.php
index edf6a5c..cbeaa70 100644
--- a/flickr_oauth_callback.php
+++ b/flickr_oauth_callback.php
@@ -5,4 +5,5 @@ unset($_GET['frob']);
 require_once(dirname(__FILE__).'/check-oauth.php');

 $token = FlickrPress::getClient()->auth_getToken($frob);
+var_dump($token); exit;
 include_once(dirname(__FILE__).'/inc.flickr_oauth_callback.php');
```

**Comment by cfletcher** [comment]
Thanks for the reply, I'll try it when I get home.
I think Flickr has disabled logins from Google OAuth recently and only allows yahoo now. That might be related.

**Comment by cfletcher** [comment]
I just get a blank page that says "NULL". There are no console errors or anything, its stuck on http://www.mydomain.com/wp-admin/admin.php?action=wpfp_flickr_oauth_callback&frob=##############-##############-######

**Comment by fukata** [comment]
Thank you debug.
Umm... Are you installed another flickr related plugins?

**Comment by cfletcher** [comment]
No, I don't have any other Flickr plugins installed and the only plugins I have enabled are below:

Adminer
Deactivate | Edit
Adminer (formerly phpMinAdmin) is a full-featured MySQL management tool written in PHP. This plugin include this tool in WordPress for a fast management of your database.

Version 1.3.2 | By Frank Bültge | View details

Blog Introduction
Deactivate | Edit
Shows an introduction before posts without messing your theme files. Configure the plugin and show your introduction.

Version 1.9.11 | By Jussi Ruokomäki | View details

Duplicate Post
Settings | Deactivate | Edit
Clone posts and pages.

Version 2.6 | By Enrico Battocchi | View details | Donate | Translate

Social Web Links
Deactivate | Edit
Display beautiful web profile icons on your widget or content. For more details, visit http://www.nischalmaniar.com/social-web-links/

Version 5.0 | By Nischal Maniar | View details

WordPress Importer
Deactivate | Edit
Import posts, pages, comments, custom fields, categories, tags and more from a WordPress export file.

Version 0.6.1 | By wordpressdotorg | View details

wp-flickr-press
Deactivate | Edit
Flickr integration for wordpress plugin.

Version 2.0.0 | By Tatsuya Fukata, Alexander Ovsov | View details

WP Post Series
Deactivate | Edit

Version 1.0.1 | By Mike Jolley | View details

WP Super Cache
Settings | Deactivate | Edit
Very fast caching plugin for WordPress.

Version 1.4.2 | By Automattic | View details

**Comment by fukata** [comment]
Thank you.
I will check with same plugins install.

**Comment by fukata** [comment]
I was checked. it's ok.
> I think Flickr has disabled logins from Google OAuth recently and only allows yahoo now. That might be related.
Your account is google account? my account is yahoo account.

**Comment by cfletcher** [comment]
I use Yahoo now but I used to use google months ago. I was just thinking the problem might be related to that.

**Comment by fukata** [comment]
Can u create new flickr user and testing?

**Comment by cfletcher** [comment]
I have the same problem with a new Flickr/Yahoo account.

**Comment by KelovpStrings** [comment]
I get the same question...It just stop an null page when you Update Token
